enhancements/update/update-blocker-lifecycle/README.md

```diff
@@ -0,0 +1,109 @@
+---
+title: update-blocker-lifecycle
+authors:
+  - "@wking"
+reviewers:
+  - "@LalatenduMohanty"
+approvers:
+  - TBD
+creation-date: 2020-09-11
+last-updated: 2021-03-30
+status: implementable
+---
+
+# Update-blocker Lifecycle
+
+We occasionally have bugs which impact update success or the stability of the target release.
+When that happens, we protect users by [removing the impacted updates from our update recommendations][graph-data-block].
+This enhancement describes the process used to identify these bugs and determine which edges should be blocked because of them.
+
+## Release Signoff Checklist
+
+- [x] Enhancement is `implementable`
+- [x] Design details are appropriately documented from clear requirements
+- [ ] Test plan is defined
```

Comment:
@xltian don't we define a test plan to proactively find/identify bugs with each release? 

If we are already doing this; then what is missing with the current process that is leading to so many issues being found / requiring us to use a blocked edge?